bool added
bool removed
string event_time
google_chat_ros/Space space
google_chat_ros/User user
